app/assets/javascripts/central.js

```diff
@@ -34,5 +36,7 @@ var Central = module.exports = {
 
       window.projectView = view;
     });
+
+    setTimeout(() => new TourController(), 200);
```

Comment:
I think it would be better to not add execution logics to the constructor. What about something like this:

```js
setTimeout(() => {
  const tourController = new TourController();
  tourController.init();
}, 200);
```

And then add an `init` method to the TourControler where you fetch the current user and then start the user?